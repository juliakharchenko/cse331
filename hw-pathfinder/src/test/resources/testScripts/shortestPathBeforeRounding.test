# Find a path with Dijkstra's algorithm where the
# shortest path between two nodes is different
# before rounding and after rounding

CreateGraph roundedGraph

AddNode roundedGraph A
AddNode roundedGraph B
AddNode roundedGraph C

AddEdge roundedGraph A B 3.77781
AddEdge roundedGraph B C 4.11675
AddEdge roundedGraph A C 7.89481

FindPath roundedGraph A C