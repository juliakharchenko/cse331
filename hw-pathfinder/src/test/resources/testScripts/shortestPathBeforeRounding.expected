# Find a path with Dijkstra's algorithm where the
# shortest path between two nodes is different
# before rounding and after rounding

created graph roundedGraph

added node A to roundedGraph
added node B to roundedGraph
added node C to roundedGraph

added edge 3.778 from A to B in roundedGraph
added edge 4.117 from B to C in roundedGraph
added edge 7.895 from A to C in roundedGraph

path from A to C:
A to B with weight 3.778
B to C with weight 4.117
total cost: 7.895